deployment.sh

```diff
@@ -69,13 +71,14 @@ for ((i = 0; i < ${#services[@]}; i++)); do
     actual_status=$(curl -m 5 -o /dev/null -Isw '%{http_code}\n' -X "$method" "https://$url")
 
     if [[ "$actual_status" -ne "$expected_status" ]]; then
-        RESULT_STRING="Health check to ${url} failed with status code: ${actual_status}"
+        RESULT_STRING="Health check to $url failed with status code: $actual_status"
+        content='🏗️: '$DEPLOYMENT_ENV$'\n'$COMMIT_TITLE_URL$'\n🚑: Health Check Failed'
 
-        content="🏗️: $DEPLOYMENT_ENV
-        $COMMIT_TITLE_URL
-        ❌: Build Failed"
+        # Use jq to construct the JSON payload properly
+        json_payload=$(jq -n --arg ERR "$RESULT_STRING" --arg CONTENT "$content" \
```

Comment:
jq is not a tool which comes out of the box. You should use something else.

Reply:
I tried to do it manually in the beginning, but it seems to be confusing and error-prone, that's why I used jq. However, check this out. It seems to be working